<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=Generator content="Microsoft Word 11 (filtered)">
<title>Autosomal Dominant and Autosomal Recessive Amelogenesis Imperfecta</title>
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:__;
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Times;
	panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face
	{font-family:AdvP3FFE8C;
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Geneva;}
@font-face
	{font-family:"\@__";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0in;
	margin-bottom:.0001pt;
	font-size:12.0pt;
	font-family:Times;}
h1
	{margin:0in;
	margin-bottom:.0001pt;
	page-break-after:avoid;
	text-autospace:none;
	font-size:12.0pt;
	font-family:Arial;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{margin:0in;
	margin-bottom:.0001pt;
	text-autospace:none;
	font-size:12.0pt;
	font-family:Arial;}
p.MsoBodyTextIndent, li.MsoBodyTextIndent, div.MsoBodyTextIndent
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:27.0pt;
	margin-bottom:.0001pt;
	text-indent:-27.0pt;
	line-height:150%;
	text-autospace:none;
	font-size:12.0pt;
	font-family:Geneva;}
@page Section1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;}
div.Section1
	{page:Section1;}
 /* List Definitions */
 ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>

</head>

<body lang=EN-US>

<div class=Section1>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:14.0pt'>Amelogenesis Imperfecta (AI)</span></b></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:6.0pt'>&nbsp;</span></p>

<p class=MsoNormal>DEFINITION:</p>

<p class=MsoNormal>DIFFERENTIAL DIAGNOSIS:</p>

<p class=MsoNormal>SIGNS &amp; SYMPTOMS:</p>

<p class=MsoNormal>ETIOLOGY:</p>

<p class=MsoNormal>DIAGNOSIS:</p>

<p class=MsoNormal>TREATMENT:</p>

<p class=MsoNormal>CASE REPORTS WITH MUTATION ANALYSES:</p>

<p class=MsoNormal style='margin-left:.25in'>Enamelin Gene (ENAM): structure</p>

<p class=MsoNormal style='margin-left:45.0pt'>Case Report 1: Mutation at exon
9/intron 9 boundary, <span style='color:black'>p.N197fsX277</span></p>

<p class=MsoNormal style='margin-left:45.0pt'>Case Report 2: Mutation in intron
6, <span style='color:black'>p.M71-Q157del</span></p>

<p class=MsoNormal style='margin-left:45.0pt'>Case Report 3: Mutation in <span
style='color:black'>exon 10</span>, <span style='color:black'>p.P422fsX448</span></p>

<p class=MsoNormal style='margin-left:45.0pt'>Case Report 4: <span
style='color:black'>Mutation in intron 8: g.6395G&gt;A, </span><span
style='color:black'>p.A158-Q178del</span></p>

<p class=MsoNormal style='margin-left:45.0pt'>Case Report 5: Mutation in exon
5, <span style='color:black'>g:2382A&gt;T, c.157A&gt;T, p.K53X</span><span
style='color:black'>.</span> </p>

<p class=MsoNormal style='margin-left:.25in;text-align:justify'>Enamelysin Gene
(MMP20): structure</p>

<p class=MsoNormal style='margin-left:45.0pt'>Case Report 1: Mutation
IVS6-2A&gt;T</p>

<p class=MsoNormal style='margin-left:.25in;text-align:justify'>Kallikrein 4
Gene (KLK4)</p>

<p class=MsoNormal style='margin-left:45.0pt'>Case Report 1: Mutation <span
style='color:#292526'>g.2142G&gt;A</span></p>

<p class=MsoNormal style='margin-left:.25in;text-align:justify'>Amelogenin Gene
(AMELX)</p>

<p class=MsoNormal style='margin-left:45.0pt'><span style='color:black'>Case
Report 1: AMELX mutation </span><span style='color:#292526'>p.W4S</span></p>

<p class=MsoNormal style='margin-left:45.0pt'><span style='color:black'>Case
Report 2: AMELX mutation </span><span style='color:#292526'>p.M1T</span></p>

<p class=MsoNormal style='margin-left:45.0pt'><span style='color:black'>Case
Report 3: AMELX mutation p.L181fsX187</span></p>

<p class=MsoNormal style='margin-left:45.0pt'><span style='color:black'>Case
Report 4: AMELX mutation p.L77H</span></p>

<p class=MsoNormal style='margin-left:45.0pt'><span style='color:black'>Case
Report 5: AMELX mutation p.P70T</span></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:14.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>DEFINITION:</span></b>
<i>Amelogenesis imperfecta</i> (AI) is a heterogeneous group of inherited
defects in dental enamel formation. The malformed enamel can be unusually thin,
soft, rough and stained. The strict definition of AI includes only those cases
where enamel defects occur in the absence of other, non oral, symptoms.
Isolated enamel malformations are caused by defects in a number of different
genes. X-linked, autosomal dominant and autosomal recessive modes of inheritance
are possible, with considerable variability in the character and appearance of
the resulting enamel. Fourteen subtypes are recognized. <i>Amelogenesis
imperfecta</i> has also been used to describe enamel defects associated with
inherited syndromes. There are over 70 such conditions. </p>

<p class=MsoNormal style='text-align:justify'>Synonyms: None</p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:10.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>DIFFERENTIAL
DIAGNOSIS:</span></b> Dental fluorosis, enamel hypoplasia secondary to systemic
disease, tetracycline staining. dentinogenesis imperfecta (DGI), AI as part of
a broader syndrome.</p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:10.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>SIGNS
&amp; SYMPTOMS:</span></b> The spectrum of enamel malformations that are
observed in patients with <i>amelogenesis imperfecta</i> is divided into three
groups based primarily upon the thickness and hardness of the dental enamel.
Differences in these parameters are believed to reflect differences in the
timing, during amelogenesis, when the disruption occurred. During tooth
formation, enamel first appears on the surface of recently deposited dentin, at
what becomes the dentino-enamel junction or DEJ. The thickness of the enamel
layer as a whole increases primarily by the elongation of enamel crystallites.
Flaws in the dentino-enamel junction can result in an enamel layer that shears
easily from the underlying dentin. Insufficient crystal elongation leaves the
enamel layer pathologically thin, or <u>hypoplastic</u>. The most severe form
of hypoplastic AI is enamel <i>agenesis</i>, where there is almost no clinical
or radiographic evidence of enamel. The teeth are yellowish brown in color,
rough in texture, and widely spaced. When the enamel crystals achieve their final
length (and the enamel layer itself achieves its final thickness), the organic
matrix separating individual enamel crystallites is degraded and resorbed. The
enamel layer then hardens or “matures” as mineral deposits on the sides of the
crystals until adjacent crystallites contact. A failure to properly remove the
organic matrix and promote the hardening of the enamel layer leads to
pathologically soft or <u>hypomaturation</u> forms of AI. The dental crowns are
of normal size and contact adjacent teeth, but the mottled, brownish-yellow
enamel is soft and has a radiodensity approaching dentin. X-linked forms of
hypoplastic and hypomaturation AI often show a distinctive phenotype in
affected females, where the enamel displays alternating vertical bands of
normal and defective enamel. This phenotype is called “Lyonization”, and
results from the alternative inactivation of either the normal or the defective
X chromosome in different cohorts of enamel forming cells. In the third type, <u>hypocalcified</u>
AI, the failure in mineralization is most extreme. The enamel layer may be of
normal thickness, but is extremely soft and wears away quickly following tooth
eruption. Patients with hypocalcified enamel form calculus rapidly and develop
acute and chronic periodontitis. </p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:10.0pt'>&nbsp;</span></p>

<p class=MsoFooter style='text-align:justify'><b><span style='font-size:14.0pt;
font-family:Times'>ETIOLOGY:</span></b><span style='font-family:Times'> <i>Amelogenesis
imperfecta</i>, by the strict definition of inherited enamel defects in the
absence of a generalized syndrome, occurs in approximately 1 out of every
14,000 people. X-linked <i>amelogenesis imperfecta</i>, which accounts for
about 5% of all cases, is caused by defects in the amelogenin gene
(Xp22.3-p22.1). Autosomal forms of AI have been shown to be caused by defects
in the enamelin (ENAM, 4q13), enamelysin (MMP20, 11q), and kallikrein 4 (KLK4,
19q) genes. Not all of the genes that cause AI have been identified. The
ameloblastin gene (AMBN, 4q) is also a prime candidate. It is likely that in
the future the classification of AI will be according to the gene involved.</span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:10.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>DIAGNOSIS:</span></b>
<i>Amelogenesis imperfecta</i> can be diagnosed by an oral examination with
dental radiographs and a family history to draw and analyze a pedigree to
determine the mode of inheritance. For information concerning how to determine
the mode of inheritance from the distribution of affected status in a pedigree,
please review:</p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:6.0pt'>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center'><u><span
style='color:black'>http://www.merck.com/mrkshared/mmanual/section21/chapter286/286b.jsp</span></u></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:6.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify'>For information concerning the
phenotypes observed in different types of amelogenesis imperfecta please
review: </p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'>Witkop
Jr. CJ, Sauk Jr. JJ (1976). Heritable defects of enamel. In: Oral Facial Genetics.
RE Stewart and GH Prescott editors. St. Louis: C.V. Mosby Co., pp. 151-226.</p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:6.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify'>Witkop Jr. CJ
(1989). Amelogenesis imperfecta, dentinogenesis imperfecta and dentin dysplasia
revisited: problems in classification. <i>J Oral Pathol </i>17:547-53.</p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='font-size:14.0pt'>TREATMENT:</span></b> Diligent oral hygiene with
frequent professional prophylactic scalings. Extensive dental restorative and
reconstructive procedures are often necessary. Hypomaturation types may respond
poorly to enamel bonding techniques.</p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>CASE
REPORTS WITH MUTATION ANALYSES:</span></b> Mutations in kindreds suffering from
autosomal dominant (ADAI) and autosomal recessive (ARAI) have been identified
in the ENAM, KLK4, and MMP20 genes. The oral phenotypes in some of these cases
are provided. It is believed that phenotype and mode of inheritance will permit
educated guesses as to which genes should be selected for mutation analyses. </p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>I.
Enamelin Gene (ENAM).</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:black'>Figure 1. Enamelin gene structure and mutations that cause
ADAI.</span></b><span style='color:black'> The structure of the human enamelin
gene showing the positions of known mutations in AI kindreds is shown at the
top. The exons are blocks numbered one through ten; the introns are lines.
Below each exon is a range of numbers indicating the amino acids it encodes.
The lower box shows the predicted effect of each mutation on the protein, the
location of the mutation in the cDNA and gene, the type of enamel defect, and
the reference(s) where each mutation are listed.</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'><u><span
style='color:black'>Reference</span></u><span style='color:black'>: Kim J-W,
Seymen F, Lin BP-L, Kiziltan B, Gencay K, Simmer JP, Hu JC-C (submitted). Two
enamelin gene mutations in kindreds with autosomal dominant amelogenesis
imperfecta. <i>J Dent Res</i>.</span></p>

<p class=MsoNormal style='text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='font-size:14.0pt;color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='font-size:14.0pt;color:black'>Case Report 1: Autosomal Dominant/Hotspot
ENAM </span></b><b><span style='font-size:14.0pt'>Mutation at the exon 9/intron
9 boundary.</span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'><span
style='color:black'>Kim J-W, Seymen F, Lin BP-L, Kiziltan B, Gencay K, Simmer
JP, Hu JC-C (submitted). Two enamelin gene mutations in kindreds with autosomal
dominant amelogenesis imperfecta. <i>J Dent Res</i>.</span></p>

<p class=MsoNormal style='text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='text-align:justify'><b>Mutation: </b>ENAM gene at the
exon 9/intron 9 boundary, g.8344delG, <span style='color:black'>p.N197fsX277</span>.
One of six Gs at the end of exon 9 was deleted, shifting the reading frame
after Gly<sup>196</sup>, and causing the synthesis of a chimeric protein
predicted to have 196 amino acids of the wild-type protein (normally 1142 amino
acids) followed by 80 novel amino acids, for a total of 276 amino acids
(p.N197fsX277). This mutation was independently identified in three unrelated
kindreds.</p>

<p class=MsoNormal style='text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='color:black'>Figure
2. </span></b><span style='color:black'>Panels C and D show frontal photograph
and a panorex radiograph for the proband taken at ages 11. Panel E through H
show intraoral photographs and a panorex radiograph taken at ages 19. The upper
anterior teeth were treated endodontically and restored with composite resin
restorations.</span></p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify'><b>Description:</b> The proband
was an 11-year-old male. Oral hygiene was poor and thick deposits of plaque and
calculus were evident, even on the facial surfaces of the anterior teeth. The
proband had an anterior openbite and the incisal edges were jagged and chipped
easily. His teeth were sensitive to thermal changes. The enamel layer was
unusually thin, with shallow horizontal grooves on the buccal middle 1/3 of
anterior teeth. The enamel appeared most normal on the occlusal surfaces of the
posterior teeth, especially near the cusp tips and on the marginal ridges, and
at the incisal line angles of the anterior teeth. Outside of these areas, the
teeth were yellow in color due to thinness. A traumatic injury to the anterior
teeth resulted in endodontic treatment of 7, 8 and 9, which further altered the
color of these teeth. Sealant and composite restorations were of normal
durability after several years of follow-up. Mutational analyses revealed that
one of seven Gs at the end of exon 9/beginning of intron 9 was deleted
(g.8344delG).</p>

<p class=MsoNormal style='text-align:justify'><b><span style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>Other
reports for this mutation:</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'>Hart
PS, Michalec MD, Seow WK, Hart TC, Wright JT (2003). Identification of the
enamelin (g.8344delG) mutation in a new kindred and presentation of a
standardized ENAM nomenclature. <i>Arch Oral Biol </i>48:589-596.</p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify'><span
style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'>Kida
M, Ariga T, Shirakawa T, Oguchi H, Sakiyama Y (2002). Autosomal-dominant
Hypoplastic Form of Amelogenesis Imperfecta Caused by an Enamelin Gene Mutation
at the Exon-Intron Boundary. <i>J Dent Res </i>81:738-742.</p>

<p class=MsoNormal style='text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:black'>Figure 3.</span></b><span style='color:black'> </span>Both
of these affected Lebanese individuals (A &amp; B) showed a generalized and
severe thinning of the enamel. The dentition of the proband (A) shows
horizontal furrows with pits while another dentition (B) appears to have been
worn smooth. <span style='color:black'>Radiographically the primary and
permanent teeth showed little to no visible enamel. The visible enamel had a
greater radiopacity compared with dentin suggesting the enamel had a greater
mineral content than dentin. Histological examination of the teeth showed the
dentin was normal. The enamel thickness did not exceed 40 µm and was often only
10—20 µm in thickness. </span><span style='font-family:AdvP3FFE8C'>This
represents a 25—100-fold reduction in enamel thickness compared with normal
(greater than 1mm). </span>In a Japanese family with this mutation the male
proband and his younger brother showed hypoplastic enamel in both their
deciduous and permanent teeth (C &amp; D) that resulted in hypersensitivity to
cold stimuli. Anterior open bite was found in both of them in the primary
dentition. Their father showed local hypoplastic enamel defects demonstrating a
horizontal lesion involving primarily the middle third of the permanent teeth
(E). In the molars, the lesion was further extended occlusally. In contrast to
his sons, he showed normal sensitivity to cold stimuli in the affected teeth.</p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>Case
Report 2: Autosomal Dominant Mutation in intron 6.</span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'><span
style='color:black'>Kim J-W, Seymen F, Lin BP-L, Kiziltan B, Gencay K, Simmer
JP, Hu JC-C (submitted). Two enamelin gene mutations in kindreds with autosomal
dominant amelogenesis imperfecta. <i>J Dent Res</i>.</span></p>

<p class=MsoNormal style='text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='text-align:justify'>Mutation: Enamelin gene mutation
(g.4806A&gt;C, IVS6-2A&gt;C), which alters the intron 6 splice acceptor site<span
style='color:black'> </span>causing ADAI. Two defective splicing outcomes are
probable for this mutation. The first is the inclusion of intron 6 (1615 bp).
This would insert multiple, in-frame stop codons preceding the most 3' exon.
Translation of this transcript would add 8 novel amino acids to 70 amino acids
of the wild-type protein, with the first 39 amino acids constituting the signal
peptide.</p>

<p class=MsoNormal style='text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='text-align:justify'><b>Description:</b> The proband
was a 12-year-old female patient in the Pediatric Dentistry Clinic at the
University of Istanbul. The proband's teeth were sensitive to thermal changes.
Oral hygiene was poor and several teeth showed cavitation from advanced dental
caries. She had rough and thin enamel in general, which allowed the dentin show
through and give the crowns a yellowish hue and horizontal grooves of severely
hypoplastic enamel. The mother had several shallow hypoplastic horizontal
grooves in the lower anterior teeth.</p>

<p class=MsoNormal style='text-align:justify'><b><span style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='color:black'>Figure
4. </span></b><span style='color:black'>Panels C through E show intraoral
photographs and a panorex of the proband’s 11 year old brother. Panel F shows
photograph of proband’s mother, taken at ages 32. Panels G through J show
intraoral photographs and a panorex of the proband, which were taken at ages
12.5.</span></p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><b><span style='font-size:14.0pt;
color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-autospace:none'><b><span style='font-size:14.0pt;
color:black'>Case Report 3: Autosomal Recessive/Dominant ENAM Mutation in Exon
10: </span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'>Hart
TC, Hart PS, Gorry MC, Michalec MD, Ryu OH, Uygur C, Ozdemir D, Firatli S, Aren
G, Firatli E (2003). Novel ENAM mutation responsible for autosomal recessive
amelogenesis imperfecta and localized enamel defects. <i>J Med Genet </i>40:900-906.</p>

<p class=MsoNormal style='text-autospace:none'><b><span style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-autospace:none'><b><span style='color:black'>Mutation:
</span></b>g.13185_13186insAG, <span style='color:black'>c.1258_1259insAG</span>,
<span style='color:black'>p.P422fsX448. </span></p>

<p class=MsoNormal style='text-autospace:none'><b><span style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify'><b>Figure 5.</b> (Top panel)
Photographs of all three probands show generalized hypoplastic enamel
characteristic of hypoplastic amelogenesis imperfecta. The enamel has fractured
from several teeth in the proband on the left. The posterior teeth are in
occlusion in all cases, and the clinically evident anterior openbite is seen in
all three probands. Bottom panel: Local circumscribed enamel pitting present in
heterozygous carriers of the ENAM g.13185_13186insAG mutation. (A) Enamel pit
present on buccal surface of mandibular incisor. (B) Enamel pit present on
buccal surfaces of maxillary incisors. (C) Enamel pit on mandibular cuspid.
Multiple enamel pits present on occlusal surface of mandibular molar (D, E) and
on maxillary molars.</p>

<p class=MsoNormal style='text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='text-align:justify'><b>Description: </b>This case is
particularly interesting because the pitting in the heterozygous condition
would not have been considered to be AI. Thus sequence variations in the genes
that participate in the etiology of AI may be cause sub-clinical defects that
could conceivably result in increased susceptibility to dental diseases, such
as caries.</p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>Case
4: </span></b><b><span style='font-size:14.0pt;color:black'>Mutation in intron
8: g.6395G&gt;A</span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'>Rajpar
MH, Harley K, Laing C, Davies RM, Dixon MJ (2001). Mutation of the gene
encoding the enamel-specific protein, enamelin, causes autosomal-dominant
amelogenesis imperfecta. <i>Hum Mol Gen </i>10:1673-7.</p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify'><b><span style='color:black'>Figure
6.</span></b><span style='color:black'> Autosomal-dominant smooth hypoplastic
AI. Photographs of proband show small, smooth, yellow teeth that result from
the enamel hypoplasia. All affected members of the family exhibited enamel
hypoplasia in both the deciduous and permanent dentitions. Clinically, the
teeth are small, thin and yellow due to a lack of enamel thickness. </span></p>

<p class=MsoNormal style='text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='margin-left:.75in;text-indent:-.75in;text-autospace:
none'><b><span style='font-size:14.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-left:.75in;text-indent:-.75in;text-autospace:
none'><b><span style='font-size:14.0pt'>Case 5:  Mutation in exon 5 (</span></b><b><span
style='font-size:14.0pt;color:black'>p.K53X) introducing a </span></b><b><span
style='font-size:14.0pt;color:black'>translation termination codon at amino
acid position 53.</span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'>Mårdh
CK, Backman B, Holmgren G, Hu JC, Simmer JP, Forsman-Semb K (2002). A nonsense
mutation in the enamelin gene causes local hypoplastic autosomal dominant
amelogenesis imperfecta (AIH2). <i>Hum Mol Gen </i>11:1069-1074.</p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='color:black'>Figure
7. </span></b><span style='color:black'>Photographs showing the clinical
manifestation of autosomal dominant local hypoplastic AI. The hypoplastic
manifestation consists of a horizontal row of pits, grooves or a large
hypoplastic area in the enamel. Apart from the variations in number and
localization of the hypoplastic defects, the phenotype was consistent within
and between the six affected families. from the same geographical area in
Västerbotten County in Northern Sweden.</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>II.
Mutations of the Enamelysin Gene.</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='font-size:14.0pt;color:black'>Disease-causing mutation in the Enamelysin
(MMP-20) Gene</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='color:black'>Figure
8.</span></b><span style='color:black'> <b>Structure of the human enamelysin
gene marking the position of the mutation</b> <b>associated with AI.</b> The
exons are indicated by the 10 vertical lines; the introns are horizontal lines.
Above each exon and below each intron are numbers indicating its length in
basepairs. Below the gene structure the 10 exons are aligned with a linear
diagram of the MMP-20 structural domains. Above the exons are numbers
indicating the range of amino acids encoded by each exon. Numbers beneath the
protein structure indicate the range of amino acids in each structural domain.</span>
At the bottom of the figure is an alignment of the human <i>MMP20</i> sequence
with the two defective sequences most likely to result from the observed splice
junction mutation. The wild type (wt) sequence is provided, beginning with the
first amino acid (Cys<sup>296</sup>) of the hemopexin domain encoded in exon 6.
A dash marks the end of the last amino acid (Arg<sup>318</sup>) encoded by exon
6. Under the wt sequence is the defective amino acid sequence <span
style='color:black'>(p.I319fs338X)</span>that would be expected by joining exon
6 to exon 8 (Ex6-8). At the bottom is the defective amino acid sequence
(p.I319X) that would be expected if intron 6 is retained (+Int6).</p>

<p class=MsoNormal style='text-align:justify'><b>&nbsp;</b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>Case
Report 1: Mutation IVS6-2A&gt;T</span></b></p>

<p class=MsoNormal style='margin-left:.25in;text-align:justify;text-autospace:
none'>Kim J-W, Simmer JP, Hart TC, Hart PS, Bartlett JD, Hu JC-C (in press).
MMP-20 mutation in autosomal recessive pigmented hypomaturation amelogenesis
imperfecta. <i>J Med Genet</i>.</p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify'><b>Description:</b> The proband
first presented at age 6 yr. in the Pediatric Dentistry Clinic at the
University of Michigan. The dental phenotype resembled the autosomal recessive
pigmented hypomaturation type of amelogenesis imperfecta.<sup>21</sup> The teeth appeared normal in size, with some crowding. The enamel layer was pigmented, showing an agar-brown discoloration. The surface of the enamel was mottled and rough, but hard and brittle. Chunks of enamel had fractured away from several teeth, most notably on the lingual of the maxillary right canine (#6), and the buccals of the mandibular left canine (#22) and right first bicuspid (#28). Radiographically, the enamel layer was usually more opaque than the underlying dentin, but not in all areas, and was never as radiopaque as normal enamel. The proband had an anterior
open bite. His teeth were generally not sensitive, except in response to
extremes in temperature. Composites were placed, but required periodic repairs
as the enamel at the restoration margins tended to chip off from the teeth. The
father of the proband was reported to be affected, and is currently edentulous
and wears maxillary and mandibular complete dentures.</p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify'><b><span style='color:black'>Figure
9. Phenotype of proband. A. </span></b><span style='color:black'>Photographs
show frontal and lateral photographs of the proband at age 12 showing composite
veneer restorations on the maxillary anterior teeth. The unrestored mandibular
anterior teeth show the mottled appearance and irregular staining. <b>B</b>.
Panorex radiograph of the same patient at age 6. The anterior openbite is evident.
<b>C</b>. Bitewing radiographs of premolars and molars at age 12 show the lack
of contrast between enamel and dentin, consistent with hypomaturation AI.</span></p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>III.
Mutations of the Kallikrein 4 Gene.</span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'><b><span
style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='font-size:14.0pt;color:black'>Case Report 1: Mutation </span></b><b><span
style='font-size:14.0pt;color:#292526'>g.2142G&gt;A</span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'>Hart
PS, Hart TC, Michalec MD, Ryu OH, Simmons D, Hong S, Wright JT (2004). Mutation
in kallikrein 4 causes autosomal recessive hypomaturation amelogenesis
imperfecta. <i>J Med Genet </i>41:545-549.</p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:black'>Description. </span></b><span style='color:#292526'>The
proband had one sibling who was also affected with a nearly identical dental
phenotype (A). Both the primary and permanent dentitions in the proband and
sibling (ages 9 and 10 years) were similarly affected showing a yellow brown
discoloration (B). The teeth were excessively sensitive to hot and cold, making
it painful to masticate. Radiographically the teeth appeared morphologically
normal in shape indicating that the enamel was of normal thickness (C). The
enamel showed only a slightly increased opacity compared with the dentin,
indicative of a decreased enamel mineral content. Enamel of the affected
children had fractured from the occlusal surfaces of the primary molars, which
is consistent with a decreased mineral content. One affected child had an
anterior dental open bite while the other child did not.</span></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:#292526'>Figure 10.</span></b><span style='color:#292526'> The family
pedigree (A) had two affected female children born to two unaffected adults.
The dentition of both affected children was normal in morphology but had a
generally yellow brown color (B) that affected both the primary and permanent
teeth. Panographic radiography (C) revealed worn and fractured enamel on the
posterior teeth and enamel that had a reduced opacity that was only slightly
greater than the underlying dentin (arrows), indicative of enamel
hypomineralization.</span></p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify'><b><span style='font-size:14.0pt'>IV.
Mutations of the amelogenin gene on the X-chromosome</span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'><b><span
style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='font-size:14.0pt;color:black'>Disease-causing mutation in the Amelogenin
(AMELX) Gene</span></b></p>

<p class=MsoNormal style='margin-right:.25in;text-align:justify;text-autospace:
none'><b>&nbsp;</b></p>

<p class=MsoNormal style='margin-right:.25in;text-align:justify;text-autospace:
none'><b>Figure 11. Structures of the human amelogenin gene on the X-chromosome
(AMELX) showing the sites of mutations known to cause X-linked AI. </b>Exons
are indicated by numbered boxes, introns by a line. The numbers below each exon
show the range of amino acids encoded by that exon. The numbers with lines
pointing to sites in the genes indicate the codon number that was affected by
mutation. A deletion mutation is indicated by “<span style='color:black'>&#8710;”.
An “X” indicates that the mutation created a stop codon.</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'><b><span
style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-right:.25in;text-align:justify;text-autospace:
none'><b><span style='color:black'>X Inactivation (Lyonization)</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><span
style='color:black'>In mammals, X chromosome expression is balanced in females </span>(XX),
<span style='color:black'>and males </span>(XY). <span style='color:black'>by
the random inactivation one of the two X chromosomes in each daughter cell
during embryonic development. As a consequence, during tooth development some
ameloblasts will express amelogenin from the maternal X-chromosome and some
will express amelogenin from the paternal X-chromosome. When AMELX is defective
on one X-chromosome in females (X*X), the enamel phenotype will be less severe
than in affected males (X*Y) and will often be manifested as alternative
vertical bands of defective and normal enamel on the crowns of the teeth. </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'><b><span
style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='font-size:14.0pt;color:black'>Case Report 1: AMELX mutation </span></b><b><span
style='font-size:14.0pt;color:#292526'>p.W4S</span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'>Kim
J-W, Simmer JP, Hu YY, Lin BP-L, Boyd C, Wright JT, Yamada CJM, Rayes SK,
Feigal RJ, Hu JC-C (2004). Amelogenin p.M1T and p.W4S mutations underlying
hypoplastic X-linked amelogenesis imperfecta. <i>J Dent Res </i>83:378-83.</p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify'><b><span style='color:black'>Figure
12.</span></b><span style='color:black'> Oral photographs of the proband (top)
and her mother (bottom) show a pattern of alternative vertical bands of normal
and hypoplastic enamel.</span></p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='font-size:14.0pt;color:black'>Case Report 2: AMELX mutation </span></b><b><span
style='font-size:14.0pt;color:#292526'>p.M1T</span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'>Kim
J-W, Simmer JP, Hu YY, Lin BP-L, Boyd C, Wright JT, Yamada CJM, Rayes SK,
Feigal RJ, Hu JC-C (2004). Amelogenin p.M1T and p.W4S mutations underlying
hypoplastic X-linked amelogenesis imperfecta. <i>J Dent Res </i>83:378-83.</p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify'><b>Description:</b> <span
style='color:black'>The teeth of the proband had sharp, thin incisal edges that
fractured easily in both the maxillary and mandibular anterior teeth. The
anterior incisors had diastemas, and the enamel layer was extremely thin and
slightly rough. The underlying dentin appeared to show through the thin enamel
covering, and gave the teeth a yellowish shade. On dental radiographs, the
enamel layer was more radio-opaque than dentin but was difficult to delineate
because of its thinness. The enamel did not appear to wear well, as some
occlusal attrition was evident. No vertical banding pattern was evident in the
enamel of affected females. The only affected male recruited from this kindred
had crowns covering all of his teeth, but his panorex from age 27 taken before
the full-mouth reconstruction, showed no evidence of enamel on any of the teeth
(based upon radiodensity and the contour of the crowns).</span></p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify'><b><span style='color:black'>Figure
13. </span></b><span style='color:black'>Oral photograph of the proband (A),
the pedigree (B), bitewing radiographs of the proband showing that the very
thin enamel layer that is only evident radiographically on the cusp tips (C,
D), DNA sequencing chromatograms (E-H) of exon 2 from an affected male member
(IV-2) showing the mutation (Mut) that changed the ATG of the wild-type (Wt)
start codon into an ACG, which is normally a threonine codon, and single
stranded conformational polymorphism (SSCP) analysis of exon 2 with arrowheads
pointing to the extra band that correlates with affection status (I).</span></p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='font-size:14.0pt;color:black'>Case Report 3: AMELX mutation p.L181fsX187</span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'>Hart
PS, Aldred MJ, Crawford PJ, Wright NJ, Hart TC, Wright JT (2002). Amelogenesis
imperfecta phenotype-genotype correlations with two amelogenin gene mutations. <i>Arch
Oral Biol </i>47:261-265.</p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-autospace:none'><b>Mutation:</b> Deletion of a
C-nucleotide exon 6 causing a frameshift alteration of the next six codons, and
a premature stop codon resulting in truncation of the protein 18 amino acids
shorter than the wild-type (g.4114delC, c.541delC, p.L181fsX187).</p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b>Description:</b>
The affected males had small teeth, with spacing and lack of contact points due
to enamel hypoplasia. The enamel present was similar in color to normal and was
hard on probing. Heterozygous females in this family had teeth of normal color
and lucency, but the contour of the enamel surface was altered by a series of
vertical grooves affecting all the teeth. Their enamel felt hard on probing and
was impenetrable to a dental explorer. This family was therefore characterized
by enamel that was of apparently normal mineral content but of reduced
thickness. </p>

<p class=MsoNormal style='text-autospace:none'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify'><b><span style='color:black'>Figure
14. </span></b>(Top) In an affected male both primary and permanent teeth have
thin, lucent enamel of uniform contour and thickness. (Bottom) A heterozygous
female: a series of alternating vertical ridges and grooves affects the
maxillary central incisors and mandibular teeth; maxillary lateral incisors are
also affected but more subtly. The enamel is of normal color and lucency. The
slightly darker grooves on the maxillary left central incisor are interpreted
as underlying dentine showing through a thin layer of enamel rather than a
mineralization defect in the enamel itself.</p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='font-size:14.0pt;color:black'>Case Report 4: AMELX mutation p.L77H</span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'>Hart
PS, Aldred MJ, Crawford PJ, Wright NJ, Hart TC, Wright JT (2002). Amelogenesis
imperfecta phenotype-genotype correlations with two amelogenin gene mutations. <i>Arch
Oral Biol </i>47:261-265.</p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-autospace:none'><b>Mutation:</b> A single
basepair change of A to causing a His to Leu change (g.3803A&gt;T, c.230A&gt;T,
<span style='color:black'>p.H77L)</span>.</p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:black'>Description:</span></b><span style='color:black'> </span>Most
of the affected males had been rendered edentulous or had had extensive
restorations, with full crowns on most teeth. Any uncrowned teeth were
yellow-brown, with loss of the normal translucency of enamel, which had also
been subject to excessive wear. Heterozygous females in family 1 had vertical
ridges and grooves on the enamel surface, with discoloration. Generally, their
teeth appeared yellow-brown, but closer examination showed that the enamel in
the grooves contributed most to the discoloration, with the enamel of the
ridges being of more normal color and lucency. The phenotype in this family was
therefore characterized clinically by enamel that was affected both in
thickness and mineralization. The predominant effect in males appeared to be an
alteration of mineral content (based on the clinical appearance of excessive
wear and marked color change), with a relatively normal enamel thickness.</p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><b>Figure 15.</b> (a) An
affected male: lower incisors are darker than the remainder of the teeth, which
have been crowned; there is moderate wear on the incisal edges of the lower
incisors. (b) A heterozygous female: incisors show vertical markings with
alternating grooves and ridges; enamel forming the grooves is darker than that
of the adjacent ridges.</p>

<p class=MsoNormal style='text-autospace:none'>&nbsp;</p>

<p class=MsoNormal style='text-autospace:none'>&nbsp;</p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='font-size:14.0pt;color:black'>Case Report 5: AMELX mutation p.P70T</span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25in;margin-bottom:0in;
margin-left:.25in;margin-bottom:.0001pt;text-align:justify;text-autospace:none'>Ravassipour
DB, Hart PS, Hart TC, Ritter AV, Yamauchi M, Gibson C, Wright JT (2000). Unique
enamel phenotype associated with amelogenin gene (AMELX) codon 41 point
mutation. <i>J Dent Res </i>79:1476-81. </p>

<p class=MsoNormal style='text-align:justify'>&nbsp;</p>

<p class=MsoNormal style='text-autospace:none'><b>Mutation:</b> A single
basepair change of A to causing a His to Leu change (g.3781C&gt;A, c.208C&gt;A,
p.P70T<span style='color:black'>)</span>.</p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b><span
style='color:black'>Description:</span></b><span style='color:black'>
Clinically, enamel in affected males did not abrade easily and was
yellow-brown, becoming opaque-white in the cervical one-fourth of the crown. In
affected males, the tooth morphology was generally normal, with enamel of
normal thickness. Radiographic examination revealed that the enamel of greater
radiopacity than dentin, indicating a relatively normal mineral content.
Females heterozygous for both the normal and mutant AMELX alleles showed a
milder expression of the condition compared with hemizygous affected males with
vertical bands of normal and affected enamel.</span></p>

<p class=MsoNormal style='text-align:justify'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;text-autospace:none'><b>Figure 16.</b>
The teeth of this 11-year-old affected male show the characteristic
yellow-brown coloration of the clinical crowns that becomes white-opaque at the
cervical region. Histological examination of thin sections from </p>

<p class=MsoNormal style='text-autospace:none'>&nbsp;</p>

</div>

</body>

</html>
